system-addon/content-src/components/ContextMenu/_ContextMenu.scss

```diff
@@ -1,5 +1,5 @@
 .context-menu {
-  background: $background-primary;
+  background: var(--newtab-background-color);
```

Comment:
Do we also need to set the text color for the context menu items? Dark theme doesn't have icons that match up with the text color anymore.. unless it's hovered:
![image](https://user-images.githubusercontent.com/438537/38014467-b59a6a7a-321d-11e8-8eb6-af32b8dd3dd1.png)
![image](https://user-images.githubusercontent.com/438537/38014490-cb124e68-321d-11e8-9b30-f840d36f1409.png)